src/main/java/touro/chess/Location.java

```diff
@@ -26,6 +28,19 @@ public String toString() {
                 '}';
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
There are a lot of tests that are comparing Location objects by their row and column, which can now be simplified by this code.